app/src/main/touchbar.js

```diff
@@ -0,0 +1,125 @@
+import {TouchBar} from 'electron';
+import plugins from './plugins';
+
+const {TouchBarButton, TouchBarPopover} = TouchBar;
+
+const aspectRatioToSize = {
+  '16:9': '1600x900',
+  '5:4': '1280x1024',
+  '5:3': '1280x768',
+  '4:3': '1024x768',
+  '3:2': '480x320',
+  '1:1': '512x512',
+  Custom: 'Custom'
+};
```

Comment:
This should be a `Map`. You could then just use `[aspectRatioToSize.keys()...]` on line 17 and `[aspectRatioToSize.values()...]` on line 40.